dask-report.html

```diff
@@ -0,0 +1,88 @@
+
+
+
```

Comment:
dask-report.html should be removed, since its just auto-generated output, I believe